### ISSUE TITLE
Eye Candy to let you know that nepho is building your cloud

### ISSUE THREAD

**Issue by tfhartmann** [huit/nepho#133]
So when nepho builds a stack, you loose your prompt and it just hangs out until it's complete, we may want to look into adding some cute little eye candy to let the use know that the stack is being deployed. 

At the moment vagrant looks like this:

``` Shell
(nepho)[alaric@dhcp-128-103-62-80 nepho]$ nepho stack create theforeman 
development 


```

Which is ok, but a bit un satisfying - we may want to add something simple like a spinning wheel and message or something like "All looks well and Nepho-bots are building a $provider Cloud for you! Please wait!" 

Ideally we'd do something  that has status to show the user, but some providers will make this easier / possible and some not at all I suspect.

**Comment by agperson** [comment]
Implemented for AWS with #215

**Comment by agperson** [comment]
Good enough for now with #221.  We can't make the Vagrant output any more useful without forking and improving the python-vagrant dependency, and I don't think that's something we want to tackle in the immediate future.
